Chart.yaml

```diff
@@ -84,3 +84,6 @@ dependencies:
     condition: global.stan.enabled
     tags:
       - stan
+  # external-es-proxy
+  - name: external-es-proxy
+    condition: global.custom_logging.enabled
```

Comment:
Why is this underscore rather than snake case?

Reply:
This one is addressed